OracleHTTPServer/README.md

```diff
@@ -1,82 +1,47 @@
-
-## Building a sample Docker Image of a OHS Domain
-To try a sample of a OHS standalone image with a domain configured, follow the steps below:
-
-Make sure you have **oracle/ohs:12.2.1-sa** image built. If not go into **dockerfiles/12.2.1** and call:
-
-        $ sh buildDockerImage.sh -v 12.2.1
+### Building Oracle JDK (Server JRE) base image
+You must first download the Oracle Server JRE binary and drop in folder `../OracleJava/java-8` and build that image. For more information, visit the [OracleJava](../OracleJava) folder's [README](../OracleJava/README.md) file.
 
-### How to Build OHS domain Image
-Go to folder **samples/1221-OHS-domain**
+        $ cd ../OracleJava/java-8
+        $ sh build.sh
 
-Run the following command:
+### Building OHS Docker Image
+IMPORTANT: You have to download the OHS binary and put it in place (see .download files inside dockerfiles/).
 
-        $ docker build --force-rm=true --no-cache=true --rm=true -t sampleohs:12.2.1 .
+Download the required package (see .download file) and drop them in the folder 12.2.1.2.0. Then go into the **dockerfiles** folder and run the **buildDockerImage.sh** script as root.
 
-Verify you now have this image in place with
+    $ sh buildDockerImage.sh -v 12.2.1.2.0
 
-        $ docker images
+IMPORTANT: The resulting image will have a  pre-configured domain. 
 
 ### How to run container
 
-**Prerequisite** : Create a docker data volume which will contain the Oracle Weblogic Proxy Plugin file
+If we want to start the OHS container without specifying any configuration for mod_weblogic:
+1. To start the OHS container with above oracle/ohs:12.2.1.2.0-sa image, run the following command:
 
-         Eg:$ docker volume create --name volume
+         docker run -it --name ohs -p 7777:7777 oracle/ohs:12.2.1.2.0-sa
 
-_This volume will be created in "/var/lib/docker" directory or the location where "/var/lib/docker" points to._
 
+If we want to start the OHS container with some pre-specified mod_weblogic configuration:
+1. Depending on your weblogic environment , create a **custom_mod_wl_ohs.conf** file by referring to container-scripts/mod_wl_ohs.conf.sample and section 2.4 @ [OHS 12c Documentation](http://docs.oracle.com/middleware/12212/webtier/develop-plugin/oracle.htm#PLGWL553)
 
-1. Depending on your weblogic environment , create a **custom_mod_wl_ohs.conf** file by referring to container-scripts/mod_wl_ohs.conf.sample and section 2.4 @ [OHS 12c Documentation](http://docs.oracle.com/middleware/1221/webtier/develop-plugin/oracle.htm#PLGWL553)
+2. Place the custom_mod_wl_ohs.conf file in a directory in the host say,"/scratch/DockerVoulme/OHSVolume" and then mount this directory into the container as a volume.
```

Comment:
Can you provide an example of the command to mount the custom_mod_wl_ohs.conf file into the container volume?